packages/appstore/src/components/wallet-header/wallet-header-title.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TWalletSvgCurrency } from 'Types';
+
+type TWalletHeaderTitle = {
+    is_demo: boolean;
+    currency: TWalletSvgCurrency;
+    jurisdiction: 'svg' | 'malta';
+};
+
+const WalletHeaderTitle = React.memo(({ is_demo, currency, jurisdiction }: TWalletHeaderTitle) => {
+    const title = is_demo ? (
+        <Text weight='bold' size='sm'>
+            <Localize
+                i18n_default_text='Demo {{currency}} Wallet'
+                values={{
+                    currency,
+                }}
+            />
+        </Text>
+    ) : (
+        <Text weight='bold' size='sm'>
+            <Localize
+                i18n_default_text='{{currency}} Wallet'
+                values={{
+                    currency,
+                }}
+            />
+        </Text>
+    );
+
+    const badge = (
```

Comment:
you can use badge component from @george-usynin-deriv badge component here: https://github.com/binary-com/deriv-app/pull/8384

Reply:
@mahdiyeh-deriv I know about this component, but it's not merged to feature branch yet and still under review. Also, JFYI, today I have troubles with router and can't connect to internet from my work laptop, so I'll continue to work only tomorrow when come to the office

Reply:
will merge the card by the EOD so you can reuse it